server/app.js

```diff
@@ -37,6 +38,57 @@ config.facebook_api_key = '2640133479605924';
 
 // let HOSTNAME = 'https://opexflow.com';
 
+// Websocket between client and server
+var http = require('http').Server(app);
+var io = require('socket.io')(http, {
+    cors: {
+        origin: '*',
+        methods: ["GET", "POST"],
+        credentials: true
+    }
+});
+io.on('connection', () =>{
+    console.log("A user is connected")
+})
+
+// Program to Monitor MySql for any chagne
+const program = async () => {
+    const connection = mysql.createConnection({
+      host: config.host,
+      user: config.username,
+      password: config.password
+    });
+  
+    const instance = new MySQLEvents(connection, {
+      startAtEnd: true // to record only the new binary logs, if set to false or you didn'y provide it all the events will be console.logged after you start the app
+    });
+  
+    await instance.start();
+  
+    instance.addTrigger({
+      name: 'monitoring all statments',
+      expression: 'opexbetadb.*', // listen to opexbetadb database !!!
```

Comment:
set opexbetadb in config, because it will be renamed.

Reply:
Done, populating DB name from config file

Reply:
why don't you use the field from the config here?

Reply:
oh, or haven't you pushed the changes yet?